### PR TITLE
chore: UX improvements

### PR DESCRIPTION
**Pull request by ralgozino**
This PR includes a series of improvements to the UX (from my POV), I'm open to discussion.

- Change VPN skip flag in create command to be consistent with the other flags.
- Improve flags and commands descriptions
- Improve some log messages
- Align description of flags between commands

### CODE REVIEW

**Review by omissis — APPROVED** [review]
SBAM

File: cmd/version.go [hunk 1]
```diff
@@ -20,7 +20,7 @@ func NewVersionCmd(versions map[string]string, tracker *analytics.Tracker) *cobr
 
 	return &cobra.Command{
 		Use:   "version",
-		Short: "Print the version number of furyctl",
+		Short: "Print the version number an build information of furyctl",
```
Comment (Al-Pragliola) [inline cmd/version.go:23]:
```suggestion
		Short: "Print the version number and build information of furyctl",
```

**Review by Al-Pragliola — CHANGES_REQUESTED** [review]
small typo

**Review by Al-Pragliola — APPROVED** [review]
lgtm

**Pull request merged**